lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -975,6 +989,23 @@ def sharez(self, other):
         self.set_zlim(z0, z1, emit=False, auto=other.get_autoscalez_on())
         self.zaxis._scale = other.zaxis._scale
 
+    def shareview(self, other):
+        """
+        Share the view angles with *other*.
+
+        This is equivalent to passing ``shareview=other`` when
+        constructing the Axes, and cannot be used if the view angles are
+        already being shared with another Axes.
+        """
+        _api.check_isinstance(maxes._base._AxesBase, other=other)
```

Comment:
Should this not be more specifically 3D Axes only?

Reply:
Good call yeah, changed this for sharez as well.